cmd/models/import.go

```diff
@@ -58,6 +68,10 @@ func NewImportCmd() *cobra.Command {
 				projectId = selected.GetCid()
 			}
 
+			if len(codeIntegrationId) == 0 {
+				codeIntegrationId = workspaceConfig.CodeIntegrationId
+			}
+
```

Comment:
add option to select code integration if you don't have workspace and code integration id